components/InputManual.js

```diff
@@ -14,23 +14,46 @@ class InputManual extends React.Component {
     return (
       <View>
         <TextInput
+          style={styles.textBox}
           placeholder = "Manually enter barcode"
           onChangeText = {(barcode) => {
             this.setState({barcode}, () => {this.props.callBackFromParent(this.state.barcode)})
             }
           }
         />
 
-         <Button
-          title="Submit number!"
+         <TouchableHighlight
+          // style={styles.buttonText}
           onPress = { () => {
             this.props.navigateToResult();
-          }}
-          />
+          }}>
+          <Text>Submit barcode</Text>
```

Comment:
text needs to be move centrally / repositioned generally(would be my preference)